packages/webex/src/calling.js

```diff
@@ -68,6 +70,67 @@ class Calling extends EventEmitter {
       });
   }
 
+  async deregister() {
+    if (!this.registered) {
+      this.log.info('Authentication: webex.internal.device.deregister already done', logContext);
+
+      return Promise.resolve();
+    }
+
+    const promises = [];
+
+    const lineIds = Object.keys(this.callingClient?.getLines());
+
+    const lines = lineIds.length ? this.callingClient.getLines() : {};
+    for (const lineId of lineIds) {
+      const line = lines[lineId];
+      if (line.getStatus() === 'active') {
+        const thisLinePromise = new Promise((resolve, reject) => {
+          line.deregister();
+          line.on('unregistered', () => {
+            this.log.info(
+              `line.deregister() successful for the line with ID: ${lineId}`,
+              logContext
+            );
+            resolve();
+          });
+          setTimeout(() => {
```

Comment:
We won't need a timeout and reject here, once a deregister is called, assume that the user doesn't care about the result anymore.

Reply:
Ah. I agree. Makes sense. Let me remove it.